app/components/jobseekers/search_results/job_alerts_link_component/job_alerts_link_component.html.slim

```diff
@@ -2,4 +2,4 @@
   .job-alert-cta__fade
   .job-alert-cta__content class="govuk-heading-s govuk-!-font-weight-regular govuk-!-margin-0" id="job-alert-cta"
     .icon.icon--left.icon--alert-white
-      = t("subscriptions.link.help_text_html", link: link_to(t("subscriptions.link.text"), new_subscription_path(search_criteria: @vacancies_search.active_criteria, origin: @origin), id: "job-alert-link-sticky-gtm"))
+      = t("subscriptions.link.help_text_html", link: link_to(t("subscriptions.link.text"), new_subscription_path(search_criteria: @vacancies_search.active_criteria, origin: @origin, coordinates_present: @vacancies_search.point_coordinates.present?), id: "job-alert-link-sticky-gtm"))
```

Comment:
I think we need a `&` before `.point_coordinates`. Try searching for a location polygon 🤯

Reply:
hmm im not seeing a problem? speak in the morning

Reply:
@alexbowen https://teaching-vacancies-review-pr-2850.london.cloudapps.digital/jobs?keyword=&location=london&job_roles%5B%5D=&phases%5B%5D=&working_patterns%5B%5D=&commit=Search